CONTRIBUTING.md

```diff
@@ -1,39 +1,46 @@
 # Contributing Guidelines
-Thank you for your interest in contributing! Please see the guidelines below before contributing and [join our "Desktop App" community channel](https://pre-release.mattermost.com/core/channels/desktop-app) to ask questions from community members and the Mattermost core team.
+Thank you for your interest in contributing! Please see the guidelines below before contributing and [join our "Developers: Desktop App" community channel](https://community.mattermost.com/core/channels/desktop-app) to ask questions from community members and the Mattermost Desktop team.
+
+You can also visit our [developer guide](https://developers.mattermost.com/contribute/desktop/) to learn more information about how to setup your environment, develop and test changes to the Desktop App.
 
 ## Issue
-Thank you for feedback. When you report a problem, please pay attention to following points.
+We really appreciate your feedback on the Desktop App. We'd ask that before you file an issue that you go through a few steps beforehand.
 
-### Does it happen on web browsers? (especially Chrome)
+### Does it reproduce in a web browser?
 Mattermost Desktop is based on Electron, which integrates the Chrome engine within a standalone application.
 If the problem you encounter can be reproduced on web browsers, it may be an issue with Mattermost server (or Chrome).
 
+If this is the case, please create an issue in the [mattermost-server](https://github.com/mattermost/mattermost-server) or [mattermost-webapp](https://github.com/mattermost/mattermost-webapp) repositories.
+
 ### Try "Clear Cache and Reload"
-It's available as `Ctrl(Command) + Shift + R`.
-Some layout problems are caused by browser cache.
-Especially, this kind of issue might happen when you have updated Mattermost server.
+Sometimes issues can be resolved simply by refreshing your Mattermost server within the app.  
+You can do this by pressing `Cmd/Ctrl+Shift+R` in the Mattermost Desktop App, or you can go to the menu > View > Clear Cache and Reload
 
 ### Write detailed information
-Detailed information is very helpful to understand the problem.
-
-For example:
+If the issue still persists, please provide detailed information to help us to understand the problem. Include information such as:
 * How to reproduce, step-by-step
 * Expected behavior (or what is wrong)
 * Screenshots (for GUI issues)
-* Application version
-* Operating system
-* Mattermost server version
+* Desktop App application version (can be viewed by going to 3-dot menu > Help, or Top Menu > Mattermost > About Mattermost on macOS)
+* Operating System
+* Mattermost Server version
 
 ## Feature idea
-Please see http://www.mattermost.org/feature-requests/ .
+If you have an idea for a new feature, we'd love to hear about it!  
+Please let us know in the Mattermost Community server by making a post in the [Feature Proposals](https://community-daily.mattermost.com/core/channels/feature-ideas) channel.
```

Comment:
is feature proposals still the way to go? we have many different places where people can ask for features that I'm a bit lost (that channel, user voice, github discussions and probably I'm missing more)

Reply:
Not sure, but I know putting them in GitHub is generally not the best way. I think directly people to Community in general makes the most sense, but I'm not completely sure it's the best way.

Reply:
You can use this URL: https://handbook.mattermost.com/contributors/contributors/ways-to-contribute#propose-new-features

It's pretty light on content, but this is something we should maintain in a single location.